action.yml

```
@@ -37,6 +37,11 @@ inputs:
     required: false
     default: "false"
 
+  minPctChange:
+    description: "Minimun percentage reduction to be committed"
+    required: false
+    default: "1"
```

Seeing as you've reported that 1% causes issues with webp, it might be worth considering changing to a higher default value. Should the value be numeric or a string?

```suggestion
    default: "2"
```

---

I'm currently using 5% and still many webp images are optimized more than once.

The underlying value is a float, but in this file all values are strings.

---

> I'm currently using 5% and still many webp images are optimized more than once.

Damn… that does not sound ideal. Early on when I built image-actions I recall that 1% was about right… but that was before it supported webp. A higher single digit default might be required. Maybe 5% is a good starting place?